.azuredevops/containerApp.yaml

```diff
@@ -39,7 +39,10 @@ properties:
         identity: {{userAssignedIdentity}} 
       - name: git-repo-org
         keyVaultUrl: https://{{appKeyVaultName}}.vault.azure.net/secrets/GIT-REPO-ORG
-        identity: {{userAssignedIdentity}}         
+        identity: {{userAssignedIdentity}}
+      - name: app-insight-key
+        keyVaultUrl: https://{{appKeyVaultName}}.vault.azure.net/secrets/APP-INSIGHT-KEY
```

Comment:
Did we agree connection string orr Managed Identity for this?

Reply:
Code updated to use MI